tronpy/proto/transaction.py

```
@@ -0,0 +1,263 @@
+
+    ref_block_bytes_hex, ref_block_hash_hex, timestamp, expiration = _get_tapos_meta(ref_block_id)
+
+    transaction_raw = tron_pb2.Transaction.raw(
+        ref_block_bytes=bytes.fromhex(ref_block_bytes_hex),
+        ref_block_hash=bytes.fromhex(ref_block_hash_hex),
+        expiration=expiration,
+        timestamp=timestamp,
+        contract=[
+            tron_pb2.Transaction.Contract(
+                type=tron_pb2.Transaction.Contract.ContractType.TransferContract,
+                parameter=google.protobuf.any_pb2.Any(
```

Pylint (3.3.7)

[error] Module 'google.protobuf.any_pb2' has no 'Any' member

(E1101)